src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+function formatNumber(num) {
+  return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+}
+
+const populationElements = document.querySelectorAll('.population');
+
+let totalPopulation = 0;
+let count = 0;
+
+populationElements.forEach((element) => {
+  const populationText = element.textContent.trim();
+  const populationNumber = parseInt(populationText.replace(/,/g, ''), 10);
```

Comment:
The code correctly trims whitespace and removes commas from the text content before parsing it as an integer. This ensures that the population numbers are accurately converted.